src/components/Blog/BlogDate.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  date: {
+    color: `${theme.palette.secondary.main}`,
+    textTransform: 'uppercase',
+    padding: theme.spacing(3, 0),
+  },
+}));
+
+const BlogDate = ({ date, text }) => {
+  const classes = useStyles();
+
+  return (
+    <div>
+      <span className={classes.date}>
+        {text}
+        {date.slice(0, 10)}

```

Comment:
Do formatowania daty możesz użyć jakiejś zewnętrznej biblioteki, często jest to https://momentjs.com/.
Jeżeli chcesz to zostawić w ten sposób to powinieneś wrzucić to do odrębnego i pliku i używać jako funkcji.

Reply:
moment dodany